RecoJets/JetProducers/python/PileupJetIDParams_cfi.py

```diff
@@ -540,36 +540,36 @@
  )
 
 ####################################################################################################################  
-simple_5x = cms.PSet(
- impactParTkThreshold = cms.double(1.) ,
- cutBased = cms.bool(False),
- tmvaWeights = cms.string("RecoJets/JetProducers/data/TMVAClassification_5x_BDT_simpleNoVtxCat.weights.xml.gz"),
- tmvaMethod  = cms.string("BDT_simpleNoVtxCat"),
- version = cms.int32(-1),
- tmvaVariables = cms.vstring(
-    "frac01",
-    "frac02",
-    "frac03",
-    "frac04",
-    "frac05",
-    "nvtx",
-    "beta",
-    "betaStar",
-    ),
- tmvaSpectators = cms.vstring(
-    "jetPt",
-    "jetEta",
-    ),
- JetIdParams = simple_5x_wp,
- label = cms.string("simple")
- )
+# simple_5x = cms.PSet(
+#  impactParTkThreshold = cms.double(1.) ,
+#  cutBased = cms.bool(False),
+#  tmvaWeights = cms.FileInPath("RecoJets/JetProducers/data/TMVAClassification_5x_BDT_simpleNoVtxCat.weights.xml.gz"),
```

Comment:
these appear to be good candidates for a cleanup.
 @ahinzmann @knash @rappoccio 
can we simply remove simple_5x, simple_5x_chs, full, simple, and PhilV0?
All of these were added in #10625 [merged Aug 20, 2015], but the training files were never added to the data files. So, from the production processing/analysis release perspective, these were not usable.